File: swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/SendMailAdvAttachments.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// A File attachment for an email
  /// </summary>
  [DataContract]
  public class SendMailAdvAttachments {
    /// <summary>
    /// (optional) Filename to specify for the attachment.
    /// </summary>
    /// <value>(optional) Filename to specify for the attachment.</value>
    [DataMember(Name="filename", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "filename")]
    public string Filename { get; set; }

    /// <summary>
    /// Contents of the attached file (must be base64 encoded)
    /// </summary>
    /// <value>Contents of the attached file (must be base64 encoded)</value>
    [DataMember(Name="data", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "data")]
    public byte[] Data { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class SendMailAdvAttachments {\n");
      sb.Append("  Filename: ").Append(Filename).Append("\n");
      sb.Append("  Data: ").Append(Data).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: swagger-client/dart/lib/model/send_mail_adv_attachments.dart
part of swagger.api;

class SendMailAdvAttachments {
  /* (optional) Filename to specify for the attachment. */
  String filename = null;
/* Contents of the attached file (must be base64 encoded) */
  String data = null;

  SendMailAdvAttachments();

  @override
  String toString() {
    return 'SendMailAdvAttachments[filename=$filename, data=$data, ]';
  }

  SendMailAdvAttachments.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    filename = json['filename'];
    data = json['data'];
  }

  Map<String, dynamic> toJson() {
    return {
      'filename': filename,
      'data': data
     };
  }

  static List<SendMailAdvAttachments> listFromJson(List<dynamic> json) {
    return json == null ? new List<SendMailAdvAttachments>() : json.map((value) => new SendMailAdvAttachments.fromJson(value)).toList();
  }

  static Map<String, SendMailAdvAttachments> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, SendMailAdvAttachments>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new SendMailAdvAttachments.fromJson(value));
    }
    return map;
  }
}
